bin/move_it.py

```diff
@@ -507,7 +512,12 @@ def __init__(self, fun, *args, **kwargs):
         self._fun = fun
 
     def process_IN_CLOSE_WRITE(self, event):
-        """On closing after writing.
+        """On closing a writable file.
+        """
+        self._fun(event.pathname)
+
+    def process_IN_MODIFY(self, event):
+        """The file was modified.
```

Comment:
This is a change in behaviour from previously, where nothing would happen until the file was closed. Maybe it should be documented ? (in the module docstring if since we don't have proper docs)

Reply:
So `EventHandler` acts on these events even if it's not in the `mask` that is set up in `main()`? Not that this is used there, so might as well remove it.

Reply:
The `process_IN_MODIFY()` is now removed.

Reply:
Sorry, missed the mask! but thanks for removing it.